util/error.go

```diff
@@ -0,0 +1,31 @@
+package util
+
+import "github.com/juju/errgo"
+
+var CouldNotSetKubectlClusterError = errgo.New("could not set cluster using 'kubectl config set-cluster'")
```

Comment:
I wouldn't export errors, just export the matchers

Reply:
But then I cannot do things here in the commands package like

```
return result, microerror.Mask(util.CouldNotSetKubectlClusterError)
```

Reply:
ah, i didn't realise this is a separate package. why not declare errors in the package they're used?

Reply:
Because I don't want to duplicate these. Something tells me I shouldn't have the same errors in several packages. (like we do in webkit projects all over :-.